stackslib/src/chainstate/burn/atc.rs

```diff
@@ -119,6 +119,20 @@ impl AtcRational {
     pub fn into_inner(self) -> Uint256 {
         self.0
     }
+
+    /// Convert to a BurnSamplePoint probability for use in calculating a sortition
+    pub fn into_sortition_probability(self) -> Uint256 {
+        // AtcRational's integer part is only 64 bits, so we need to scale it up so that it occupes the
+        // upper 64 bits of the burn sample point ranges so as to accurately represent the fraction
+        // of mining power the null miner has.
```

Comment:
Is this comment accurate, shouldn't it be:

```suggestion
        // AtcRational's fractional part is only 64 bits, so we need to scale it up so that it occupies the
        // upper 64 bits of the burn sample point ranges so as to accurately represent the fraction
        // of mining power the null miner has.
```

Reply:
Fixed